### PR TITLE
Fix: POI of associated images is not updated in NIJS on correction. [SDESK-4871]

### PR DESCRIPTION
**Pull request by GyanP**
Handle media updates on correction so that multiple crops can be generated for associated media as well.

### CODE REVIEW

File: features/content_publish.feature [hunk 1]
```diff
@@ -3544,7 +3555,7 @@ Feature: Content Publishing
                             "media": "600x800_new",
                             "mimetype": "image/jpeg"
                         },
-                        "1280x720": null
+                        "1280x720": "__none__"
```
Comment (petrjasek) [inline features/content_publish.feature]: shouldn't it be actually populated now?

Comment (GyanP) [reply features/content_publish.feature]: Not actually, because we do flush the renditions.

Comment (petrjasek) [reply features/content_publish.feature]: ok, so how does it test the issue?

Comment (GyanP) [reply features/content_publish.feature]: When we do send a correction with updated poi the updated custom crops weren't generated earlier. But now we do generate the updated custom crops and if there are any corps that aren't defined in vocabulary if will throw an error. That's why this test was failing after current changes. So, it covers both the scenarios the one it was originally written for and also the current one. And that's why I thought there is no need to write a separate test for the current fix.

Comment (petrjasek) [reply features/content_publish.feature]: I see, ok then

**Pull request merged**